CONTRIBUTING.md

```diff
@@ -153,10 +153,25 @@ It is important to note that, for authorship, these should be substantial intell
 
 ## Peer review
 
-All pull requests will undergo peer review.
-Participants in this project should review pull requests, which can be done using [GitHub's review interface](https://help.github.com/articles/about-pull-request-reviews/ "GitHub: about pull request reviews").
-They should suggest modifications or, potentially, directly edit the pull request to make suggested changes.
-As a reviewer, it's helpful to note the type of review you performed: did you look over the source code, did you run the source code, did you look at and interpret the results or a combination of these?
+All pull requests will undergo peer review. The review intends to ensure that a pull request conforms to the following basic quirements.
+
+- The code is correct.
+- The results can be reproduced in a reasonable amount of time.
+- The results are expected.
+- The documentation describes the purpose, methods, results, input, output, and how to run the code.
+- The code follows basic style guidelines.
+- The code contains comments that explain non-obvious procedures.
+
+Participants in this project should review pull requests, which can be done using [GitHub's review interface](https://help.github.com/articles/about-pull-request-reviews/ "GitHub: about pull request reviews"). Following are the basic guidelines for reviewing pull requests:
+
+- Note the type of review you performed: did you look over the source code, did you look over the documentation, did you run the source code, did you look at and interpret the results or a combination of these?
+- Suggest modifications or, potentially, directly edit the pull request to make suggested changes, so that the pull request conforms to the aforementioned basic requirements.
```

Comment:
```suggestion
- Suggest modifications or, potentially, directly suggest, _but do not commit changes to_, the pull request, so that the pull request conforms to the aforementioned basic requirements.
```